unittesting/package.py

```diff
@@ -92,6 +92,8 @@ def unit_testing(self, stream, package, settings, cleanup_hooks=[]):
                         raise Exception("`legacy_runner=True` is deprecated.")
                     testRunner = DeferringTextTestRunner(
                         stream, verbosity=settings["verbosity"], failfast=settings['failfast'])
+                    if "condition_timeout" in settings:
```

Comment:
"condition_timeout" is always in settings, as it is defined in `DEFAULT_SETTINGS`.
You could use remove all references of `DEFAULT_CONDITION_TIMEOUT` and let `DEFAULT_SETTINGS` define the default.

Reply:
Clear, I can do `testRunner.condition_timeout = settings["condition_timeout"]` in here, but I'm afraid the 2 `DeferringTextTestRunner` classes still need to define a default though, in case they are imported directly / subclassed.

...Or should I use `UnitTestingMixin.load_unittesting_settings` from within  `DeferringTextTestRunner`?

Reply:
Updated code to use `DEFAULT_SETTINGS`